### PR TITLE
Fix flake-pilot build errors

### PR DESCRIPTION
**Pull request by AymenOtt**
Building flake-pilot using spec file does not work.
- New common directory missing in the Makefile
- New config template naming updated
- Error related anstyle-parse, does not build with latest version of anstyle-parse > 0.2.1
- Subcommand flake-ctl podman is not available due to missing bins

### CODE REVIEW

**Review by isbm — APPROVED** [review]
LGTM

File: Makefile [hunk 1]
```diff
@@ -111,6 +112,8 @@ install:
 		$(DESTDIR)$(TEMPLATEDIR)/firecracker.json
 	install -m 644 doc/*.8 ${DESTDIR}/usr/share/man/man8
 	install -m 755 utils/* $(DESTDIR)$(SBINDIR)
+	install -m 644 common/tests/data/cfg-v1/* \
```
Comment (Ichmed) [inline Makefile:115]: Why is this taken from a test directory? That can not be right.

If this file is required it needs to be provided from somewhere else.

Comment (isbm) [reply Makefile:115]: Actually yes, @AymenOtt. Why tests are installed?

**Review by isbm — CHANGES_REQUESTED** [review]
Test templates are installed?

File: Makefile (hunk 1 above)
Comment (AymenOtt) [reply Makefile:115]: those are templates configs files , there were before in /flake-ctl/template and completely disappeared in the latest version , the only location is under common/test/data/cfg with different names

Comment (Ichmed) [reply Makefile:115]: could you please copy them to `flake-ctl/flake-ctl-podman/templates` and `podman-ctl-firecracker` respectively and install them from there if they are still needed? You should never install from a test

Comment (AymenOtt) [reply Makefile]: agree ! Done

File: flake-ctl/flake-ctl-firecracker/templates/firecracker.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Example configuration, version 1
+#
+vm:
+  name: dragonbomb
+  target_app_path: /highway/to/hell
```
Comment (isbm) [inline flake-ctl/flake-ctl-firecracker/templates/firecracker.yaml:5]: @AymenOtt wonderful.

**Review by isbm — CHANGES_REQUESTED** [review]
In a package that part might be a bit adjusted 😉

**Review by isbm — APPROVED** [review]
@Ichmed LGTM, maybe you can say smth, otherwise pls merge.

File: flake-ctl/flake-ctl-firecracker/src/defaults.rs [hunk 3]
```diff
@@ -26,7 +26,7 @@ pub const FLAKE_DIR: &str =
 pub const FIRECRACKER_PILOT: &str =
     "/usr/bin/firecracker-pilot";
 pub const FLAKE_TEMPLATE_FIRECRACKER:&str =
-    "/etc/flakes/firecracker-flake.yaml";
+    "/etc/flakes/firecracker.yaml";
```
Comment (isbm) [inline flake-ctl/flake-ctl-firecracker/src/defaults.rs:29]: @Ichmed I am still wondering why the general config is not used here?

**Review by Ichmed — APPROVED** [review]

File: flake-ctl/flake-ctl-firecracker/src/defaults.rs (hunk 3 above)
Comment (Ichmed) [reply flake-ctl/flake-ctl-firecracker/src/defaults.rs:29]: Just hasn't been touched since rhen

**Comment by Ichmed** [comment]
Fixes #189

**Pull request merged**